girderformindlogger/api/v1/user.py

```diff
@@ -812,19 +812,21 @@ def getUsersDetails(self):
         .errorResponse(('You do not have write access for this user.',
                         'Must be an admin to create an admin.'), 403)
     )
+
     def updateUser(
         self,
         user,
+        login=None,
         displayName="",
         email="",
         admin=False,
         status=None,
-        firstName=None,
-        lastName=None
+        firstName=None
     ): # 🔥 delete firstName and lastName once fully deprecated
-        user['firstName'] = displayName if len(
-            displayName
-        ) else firstName if firstName is not None else ""
+
+        login = cherrypy.request.body.params['login']
+        user['firstName'] = firstName if firstName is not None else ""
+        user['login'] = login if login is not None else user['login']
```

Comment:
We're [phasing out `userId` visibility](https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/153), so rather than adding this functionality to an already deprecated function that requires `userId`, we should make or update a call that relies on a user being logged in and modifies only that user. See https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/635.